### DISCUSSION TITLE
Styling

### DISCUSSION THREAD

**Discussion by eekhohq** [caronchen/obsidian-epub-plugin#4] [Ideas]
Hey, it would be great if we could have CSS styling added for the epub reader mode, or at least some form of darkmode. 
Loving the plugin so far!

**Comment by gark81** [comment]
That's a great idea

in light mode epub reader works well, but when you are in dark mode, epub is hard to view, and index so difficult to read
if epub reader could adapt to dark mode, it will be the best for that!
